### ISSUE TITLE
Vignette will not load

### ISSUE THREAD

**Issue by rgzn** [dbspitz/migrateR#1]
The vignette does note load with the built in vignette() function. I can still access it by cloning the repo and manually opening the page. 

After installing from github:
`> vignette("migrateR")
Warning message:
vignette ‘migrateR’ not found `

**Comment by dbspitz** [comment]
Thanks rgzn,

Found two issues here: a problem with one of the vignette's plots and insufficient install instructions; "install_github" also needs the optional argument "build_vignettes = T". Fixed the plot and updated the install instructions. Also added a couple quick fixes to improve the plot methods. Keep me posted if you run into any other issues!

**Comment by bklaver49** [comment]
I tried to install migrateR with the command
install_github("dbspitz/migrateR/migrateR", build_vignettes = T) 
and received an error message that htmltools were not compatible.  I was using version R-3.5.1.  I upgraded to version R-3.6.1 and made sure that htmltools were upgraded.  I still receive the same error that htmltools were not compatible and vignettes did not load.  I then ran the command without vignettes=T and the package loaded just fine but no vignettes - as expected.
